ci/run_tests.sh

```diff
@@ -20,7 +20,11 @@ if [[ $(uname) == "Linux" && -z $DISPLAY ]]; then
     XVFB="xvfb-run "
 fi
 
-PYTEST_CMD="${XVFB}pytest -m \"$PATTERN\" -n auto --dist=loadfile -s --strict --durations=10 --junitxml=test-data.xml $TEST_ARGS $COVERAGE pandas"
+if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+    PYTEST_CMD="${XVFB}pytest -m \"$PATTERN\" --durations=10 --junitxml=test-data.xml $TEST_ARGS $COVERAGE pandas";
+else
+    PYTEST_CMD="${XVFB}pytest -m \"$PATTERN\" -n auto --dist=loadfile -s --strict --durations=10 --junitxml=test-data.xml $TEST_ARGS $COVERAGE pandas"
```

Comment:
We surely don't want different pytest calls. Not sure why this should be different than for the rest of builds, but we've got `$TEST_ARGS` to define build specific parameters.

This file is generic, it should be untouched in this PR.

Reply:
@datapythonista , 

Thanks for the feedback. All flags are not supported for pytest command in arm64. Command "**xvfb-run pytest -m "(not slow and not network and not clipboard)" -n auto --dist=loadfile -s --strict --durations=10 --junitxml=test-data.xml pandas**" when tested locally on arm64 platform, it is giving below error:
**ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --strict-data-files -n --dist=loadfile pandas**

So, I have removed these flags for arm64 jobs. If we have to use $TEST_ARGS, then I suppose I have to add this in every x86_64 job or define specifically in travis file with the platform conditional clause.

Please suggest if we can use $TEST_ARGS differently.

Reply:
`--strict-data-files` is from pandas, in `pandas/conftest.py`. I don't see why it wouldn't be recognized by pytest, but we need to have it. Are you calling pytest from a different directory?

Reply:
> `--strict-data-files` is from pandas, in `pandas/conftest.py`. I don't see why it wouldn't be recognized by pytest, but we need to have it. Are you calling pytest from a different directory?

@TomAugspurger , I am running pytest from same directory. I am not sure why I am getting this issue. Do I need to add any extra dependendy?

Reply:
I wouldn't think so.

The `-n --dist` comes from pytest-xdist. Is that installed?

Reply:
Yep, the issue is the missing dependency. To test locally, you should have a conda environment named `pandas-dev` created by running `conda env create` from the root of the repo. If you already have that, run `conda env update` to install the missing dependencies.

Reply:
> I wouldn't think so.
> 
> The `-n --dist` comes from pytest-xdist. Is that installed?

I have verified logs and pytest-xdist is installed. It is being displayed(pytest-xdist              1.31.0                     py_0) in **conda list** . I have tested again in travis with all flags mentioned in pytest and I am facing below error:
"xvfb-run pytest -m "(not slow and not network and not clipboard)" -n auto --dist=loadfile --durations=10 --junitxml=test-data.xml pandas
============================= test session starts ==============================
platform linux -- Python 3.7.6, pytest-5.3.5, py-1.8.1, pluggy-0.12.0
rootdir: /home/travis/build/ossdev/pandas, inifile: setup.cfg, testpaths: pandas
plugins: xdist-1.31.0, forked-1.1.3, hypothesis-5.5.4
gw0 I / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw18 I / gw19 I / gw20 I / gw21 I / gw22 I / gw23 I / gw24 I / gw25 I / gw26 I / gw27 I / gw28 I / gw29 I / gw30 I / gw31 I
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself."

Please suggest if I am missing anything. Thanks.

Reply:
Can you merge master and push again. Looks like a problem in the azure systems, nothing to do with your changes. Relaunching the build should fix it.

Reply:
> Can you merge master and push again. Looks like a problem in the azure systems, nothing to do with your changes. Relaunching the build should fix it.

Do I need to remove the platform check added for running pytest command and then merge into master or Do I need to add other changes you suggested without removing platform check in the run_tests.sh and push the commit? Can you please confirm.

Reply:
can u fix this

Reply:
As mentioned in comment https://github.com/pandas-dev/pandas/pull/30641#discussion_r388227224, I am facing timeout issues if I added the flags and I have removed xvfb-run as I am not running any clipboard tests and need to install xvfb separately if I need to add it.

Reply:
This is incorrect, and we're not going to have different pytest calls depending on the architecture. So let's fix this first, and if there are any problems, we'll have a look and find a solution.

Reply:
@datapythonista ,

I have added commit with suggested changes. Please review.